### ISSUE TITLE
Please enable jack in the build

### ISSUE THREAD

**Issue by ericfont** [flathub/org.shotcut.Shotcut#4]
Just tried out the flatpak build and found jack doesn't work, presumably because it wasn't enabled in the build.  Jack is an advertised feature of shotcut, so it would be nice if the flatpak has jack enabled.

**Comment by TingPing** [comment]
I don't think it is a feature Flatpaks should get accustomed to using. Pulseaudio (or pipewire) is the supported audio path now and into the future.

**Comment by nedrichards** [comment]
@eszlari absolutely. JACK use cases are certainly within the Pipewire scope.

**Comment by TingPing** [comment]
But who knows how long until Pipewire is viable.

**Comment by ericfont** [comment]
@eszlari ok, I tried afte rupdating and jack works...provided that the video I play is same sample rate as what jack is configured for.  I do get a crash if the samples rates don't match...that crash only seems to happen in the flatpak, but not from Arch User Repo's shotcut-bin.  So despite that, I'd consider the problem resolved, cause someone using jack would probably want to use identical sample rates anyways.

**Comment by ericfont** [comment]
command line output: [shotcut-crash.txt](https://github.com/flathub/org.shotcut.Shotcut/files/1986377/shotcut-crash.txt)

**Comment by ericfont** [comment]
I also got a variant crash: [shotcut-alternative-crash.txt](https://github.com/flathub/org.shotcut.Shotcut/files/1986397/shotcut-alternative-crash.txt)

**Comment by eszlari** [comment]
1. Clone the json file
2.  in the `mlt` module: 
- add `"--enable-debug"` under `"config-opts"` 
- add `"build-options": "no-debuginfo=true"`
3. Compile with:
```
$ flatpak-builder --install build_dir org.shotcut.Shotcut.json
```
4. post the full backtrace
